src/styles/theme/defaultTheme.ts

```diff
@@ -461,9 +461,14 @@ export const defaultTheme = createTheme({
       styleOverrides: {
         root: ({ theme }) => ({
           borderRadius: '24px',
-          '.MuiInputBase-root > :first-child': {
+          '.MuiInputBase-root': {
             // Prevent first child's edges overflowing due to border-radius
-            marginLeft: theme.spacing(4),
+            '.MuiInputAdornment-positionStart': {
+              marginLeft: theme.spacing(4),
+            },
+            '.MuiInputBase-input:not(.MuiInputBase-inputAdornedStart)': {
```

Comment:
Is it possible to avoid or limit custom css selectors? I did not check this, but something like `root: ({ theme }) => ({input: {},` ?

Reply:
Doesn't seem so